### PR TITLE
Moved to actions

### PR DESCRIPTION
**Pull request by evaherrada**

### CODE REVIEW

File: library.properties [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-version=1.3.0
+version=1.4.0
 author=Adafruit
 maintainer=Adafruit<info@adafruit.com>
 sentence=Tiny LoRa Library for TTN
 paragraph=Small LoRa library for use with The Things Network
 category=Communication
 url=https://github.com/adafruit/TinyLoRa/
 architectures=*
+depends=DHT sensor library
```
Comment (brentru) [inline library.properties:10]: @dherrada This library doesn't have a dep. on the DHT sensor lib.?

File: .github/workflows/githubci.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - name: test platforms
+      run: python3 ci/build_platform.py leonardo zero
+
+    - name: clang
+      run: python3 ci/run-clang-format.py -e "ci/*" -e "bin/*" -r . 
+
+    - name: doxygen
+      env:
+        GH_REPO_TOKEN: ${{ secrets.GH_REPO_TOKEN }}
+        PRETTYNAME : "Adafruit ILI9341 Display Library"
```
Comment (brentru) [inline .github/workflows/githubci.yml]: @dherrada  name is wrong here, should be "Adafruit TinyLoRa Library"

Comment (evaherrada) [reply .github/workflows/githubci.yml]: @brentru Thanks for catching that

File: library.properties (hunk 1 above)
Comment (evaherrada) [reply library.properties:10]: I think that the library doesn't, but some of the examples do. Is there a way to add it in a different spot so it shows up as a dependency for the examples but not the library itself?

Comment (brentru) [reply library.properties:10]: I don't think so. Just looked at Adafruit IO Arduino which has a lot of examples using libraries. Adds them to `depends` same as here (https://github.com/adafruit/Adafruit_IO_Arduino/blob/master/library.properties#L10).

**Review by brentru — APPROVED** [review]

**Pull request merged**